AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb

```diff
@@ -78,32 +72,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Normally we wouldn't need to think about this too much, but IPython/Jupyter notebooks behave a touch differently than \"normal\" python.\n",
-    "\n",
-    "Inside of IPython, it's often easiest to use the Jupyter ``nbagg`` or ``notebook`` backend.  This allows plots to be displayed and interacted with in the browser in a Jupyter notebook.  Otherwise, figures will pop up in a separate GUI window.\n",
-    "\n",
-    "We can do this in two ways:\n",
-    "\n",
-    "1. The IPython ``%matplotlib backend_name`` \"magic\" command (or ``plt.ion()``, which behaves similarly)\n",
-    "   - Figures will be shown automatically by IPython, even if you don't call ``plt.show()``.\n",
-    "     \n",
-    "2. ``matplotlib.use(\"backend_name\")``\n",
-    "   - Figures will only be shown when you call ``plt.show()``.\n",
-    "\n",
-    "Here, we'll use the second method for one simple reason: it allows our code to behave the same way regardless of whether we run it inside of an Jupyter notebook or from a python script at the command line.  Feel free to use the ``%matplotlib`` magic command if you'd prefer.\n",
+    "If you are working interactively at an (I)python prompt, the GUI framework is not critical (mostly aesthetic) however when working in Jupyter we need to pick a backend that integrates with Jupyter (javascript) framework.\n",
     "\n",
-    "One final note: You will always need to do this before you ``import matplotlib.pyplot as plt``."
+    "To select the backend use ``matplotlib.use(\"backend_name\")``, in this case we want ``'nbagg'``\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "matplotlib.use('nbagg')"
+    "matplotlib.use('nbagg')\n",
+    "print(matplotlib.get_backend())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "which must be done *before* you `import matplotlib.pyplot as plt`.\n",
+    "\n",
+    "You can also set the backend via a 'magic ``%matplotlib backend_name``.  In addition to setting the backend, the magic also calls `plt.ion()`, which puts Matplotlib in 'interacitve mode' (the inverse is `plt.ioff()`).  In 'interactive mode' figures are shown (injected into the web page in the notebook) as soon as they are created.  Otherwise, figures are not shown until you explicitly call `plt.show()`.\n",
```

Comment:
missing closing quote for "magic". Also, we should probably make it clear that this is an "ipython magic".